explorer/db/ent/schema/msg.go

```diff
@@ -24,54 +23,36 @@ type Msg struct {
 // Fields of the XMsg.
 func (Msg) Fields() []ent.Field {
 	return []ent.Field{
-		field.UUID("UUID", uuid.UUID{}).
-			Default(uuid.New),
-		field.Int("Block_ID").
-			Optional(),
-		field.Bytes("SourceMsgSender").
-			MaxLen(20),
-		field.Bytes("DestAddress").
-			MaxLen(20),
-		field.Bytes("Data"),
-		field.Uint64("DestGasLimit"),
-		field.Uint64("SourceChainID"),
-		field.Uint64("DestChainID"),
-		field.Uint64("StreamOffset"),
-		field.Bytes("TxHash").
-			MaxLen(32),
-		field.Bytes("BlockHash").
-			MaxLen(32).
-			Optional(),
-		field.Uint64("BlockHeight").
-			Optional(),
-		field.Bytes("ReceiptHash").
-			MaxLen(32).
-			Optional(),
-		field.String("Status").
-			Optional().
-			Default("PENDING"),
-		field.Time("BlockTime").
-			Optional(),
-		field.Time("CreatedAt").
-			Default(time.Now()),
+		field.Bytes("sender").MaxLen(20),
+		field.Bytes("to").MaxLen(20),
+		field.Bytes("data"),
+		field.Uint64("gas_limit"),
+		field.Uint64("source_chain_id"),
+		field.Uint64("dest_chain_id"),
+		field.Uint64("offset"),
+		field.Bytes("tx_hash").MaxLen(32),
+		field.Bytes("receipt_hash").MaxLen(32).Optional(),
+		field.String("status").Optional().Default("PENDING"),
+		field.Time("created_at").Default(time.Now()),
 	}
 }
 
 // Indexes of the Msg.
 func (Msg) Indexes() []ent.Index {
 	return []ent.Index{
-		index.Fields("SourceChainID", "DestChainID", "StreamOffset", "Block_ID"),
+		index.Fields("sender"),
+		index.Fields("to"),
+		index.Fields("status"),
+		index.Fields("tx_hash"),
```

Comment:
Can there be more than one XMsg with the same `tx_hash`? Should this field also be unique?